src/app/inventory/ConnectedInventoryItem.tsx

```diff
@@ -30,7 +31,7 @@ interface StoreProps {
   inventoryCuratedRoll?: InventoryCuratedRoll;
 }
 
-function mapStateToProps(state: RootState, props: ProvidedProps): StoreProps {
+function mapStateToProps(state: RootState, props: ProvidedProps): Props {
```

Comment:
This should always return only `StoreProps` - that interface is meant to represent the subset of props which are returned from `mapStateToProps`.